cosi.proto

```diff
@@ -116,7 +116,13 @@ message ProvisionerCreateBucketRequest {
 }
 
 message ProvisionerCreateBucketResponse {
-    // Intentionally left blank
+    // This field is REQUIRED
+    string bucket_name = 1;
+
+    // This field is OPTIONAL
+    string bucket_endpoint = 2;
+
+    map<string,string> protocol_parameters = 3;
```

Comment:
Please specify if this is optional, is it expected to be nil?

Reply:
It could be optional depending on the provider.  This is the data that will be filled in in the b.spec.protocol fields and is what will be needed by the pod to talk to the bucket in the provider.  I'll mark it as optional and try to give some context.

Reply:
Please mark it Optional

Reply:
Done